hydra_base/lib/scenario.py

```diff
@@ -19,6 +19,7 @@
 
 import logging
 import six
```

Comment:
Not in this commit, but the `six` import is now unused.

There are probably many such instances from the 2-to-3 transition era, so maybae a separate issue for me to remove `six` references.